### PR TITLE
GLTFExporter: Clean up.

### PR DESCRIPTION
**Pull request by linbingquan**
I am not sure this PR will be better. Please to review the code.

### CODE REVIEW

File: examples/js/exporters/GLTFExporter.js [hunk 1]
```diff
@@ -1777,7 +1777,7 @@ THREE.GLTFExporter.prototype = {
 
 				}
 
-				if ( ! equalArray( object.matrix.elements, [ 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1 ] ) ) {
+				if ( ! equalArray( object.matrix.elements, new THREE.Matrix4().toArray() ) ) {
```
Comment (Mugen87) [inline examples/js/exporters/GLTFExporter.js:1780]: In this case why not:
```js
if ( object.matrix.equals( THREE.Matrix4() ) === false ) ) {
```

Comment (mrdoob) [reply examples/js/exporters/GLTFExporter.js:1780]: Or maybe just just this somewhere in `GLTFExporter`:

```js
const identityArray = [ 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1 ];

function isIdentityMatrix( matrix ) {

    return equalArray( matrix.elements, identityArray );

}
```

**Review by Mugen87 — APPROVED** [review]

File: examples/js/exporters/GLTFExporter.js [hunk 2]
```diff
@@ -1777,7 +1791,7 @@ THREE.GLTFExporter.prototype = {
 
 				}
 
-				if ( ! equalArray( object.matrix.elements, [ 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 1 ] ) ) {
+				if ( ! isIdentityMatrix( object.matrix ) ) {
```
Comment (mrdoob) [inline examples/js/exporters/GLTFExporter.js:1794]: Sorry, one last change.

```js
if ( isIdentityMatrix( object.matrix ) === false ) {
```

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**